src/main/jte/auth/login.jte

```diff
@@ -26,24 +26,24 @@
                 <span>${context.localizeOrDefault(message, message)}</span>
             </div>
         @endif
-        <a class="input btn btn-primary" href="/oauth2/authorization/kit">${context.localize("login.kit")}</a>
+        <a class="input btn btn-primary" href="/oauth2/authorization/kit" aria-label="${context.localize("login.kit_login")}" title="${context.localize("login.kit_login")}">${context.localize("login.kit")}</a>
         @if(guestEnabled)
-            <a class="input btn btn-secondary" href="/login/guest">${context.localize("login.guest")}</a>
+            <a class="input btn btn-secondary" href="/login/guest" aria-label="${context.localize("login.guest_login")}" title="${context.localize("login.guest_login")}">${context.localize("login.guest")}</a>
         @endif
         <br>
         <details class="collapse collapse-plus">
             <summary class="collapse-title font-medium">${context.localize("login.admin")}</summary>
             @template.util.form(csrf = csrf, cssClass = "collapse-content fieldset", action = "/login", method = POST, content = @`
                 <input id="username" name="username" type="hidden" value="admin"/>
                 <label class="fieldset-label" for="password">${context.localize("login.password")}</label>
-                <input id="password" name="password" type="password" placeholder="Passwort" class="input" required/>
+                <input id="password" name="password" type="password" placeholder="${context.localize("login.password")}" class="input" required/>
 
                 <label class="fieldset-legend text-xs items-center">
-                    <input id="remember-me" name="remember-me" type="checkbox" class="toggle" checked />
+                    <input id="remember-me" name="remember-me" type="checkbox" class="toggle" checked aria-label="${context.localize("login.remember")}" title="${context.localize("login.remember")}"/>
```

Comment:
This text is already the text of the input, I don't think repeating it is necessary.